### PR TITLE
add position to toast

### LINKED ISSUE #2160: Toast - add right aligned example
Add example of toast where it is aligned to the right instead of center.

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-2301--keen-mayer-a86c8b.netlify.app/templates/toast-notifications?q=toast)

#### What does this PR do?
Added a example of Toast with using passing in `position` to Toast to give an example to match our guidance for the placement section
#### Where should the reviewer start?
ToastNotificationPositionExample.js
#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [x] Visual snapshots are reasonable

#### What are the relevant issues?
closes #2160 
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
 backwards compatible

### CODE REVIEW

File: aries-site/src/examples/templates/toast-notifications/ToastNotificationPositionExample.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          of the page.
+        </Paragraph>
+        <Button label="Show the Toast Notification" onClick={onOpen} primary />
+      </Box>
+      <Box align="center" gap="small">
+        {visible && (
+          <Notification
+            toast={{ position: 'top-right' }}
+            status="normal"
+            title="This toast notification is aligned to be top-right"
```
Comment (taysea) [inline aries-site/src/examples/templates/toast-notifications/ToastNotificationPositionExample.js:24]:
```suggestion
            title="This toast notification is aligned to be top-right."
```

File: aries-site/src/pages/templates/toast-notifications.mdx [hunk 2]
```diff
@@ -132,6 +133,13 @@ Follow these placement guidelines for Toast Notifications:
 - Reminder that toasts should not be used to convey a batch of information as this would quickly hinder their visibility and cause notification fatigue.
 - Consider incorporating a Notification Center where a user can access full information about tasks and asynchronous events/alerts.
 
+
```
Comment (taysea) [inline aries-site/src/pages/templates/toast-notifications.mdx:136]:
```suggestion
### Top-right Aligned Toast
```

File: aries-site/src/pages/templates/toast-notifications.mdx [hunk 3]
```diff
@@ -132,6 +133,13 @@ Follow these placement guidelines for Toast Notifications:
...
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/toast-notifications/ ToastNotificationPositionExample.js"
```
Comment (taysea) [inline aries-site/src/pages/templates/toast-notifications.mdx:138]:
```suggestion
  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/toast-notifications/ToastNotificationPositionExample.js"
```

File: aries-site/src/pages/templates/toast-notifications.mdx (hunk 2 above)
Comment (taysea) [reply aries-site/src/pages/templates/toast-notifications.mdx:136]: Just a suggestion to call out a little more that this example is for top-right aligned toast.

Comment (britt6612) [reply aries-site/src/pages/templates/toast-notifications.mdx:136]: I like that I was actually thinking about adding this as well when I put it up

**Review by halocline — APPROVED** [review]

**Review by taysea — APPROVED** [review]

**Pull request merged**